url_screenshot/icon_url_screenshot/connection/connection.py

```diff
@@ -0,0 +1,18 @@
+import komand
+from .schema import ConnectionSchema
+# Custom imports below
+
+
+class Connection(komand.Connection):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(input=ConnectionSchema())
+
+    def connect(self, params):
```

Comment:
if no connection, have the function `pass`

Reply:
oops - yeah, done.